compiler/passes/filesToAST.cpp

```diff
@@ -220,3 +239,49 @@ static void gatherWellKnownTypes() {
     }
   }
 }
+
```

Comment:
I was copying the style of gatherWellKnownTypes. I don't really care what these variables are called and would personally prefer to put the string literals into the error message calls directly (and not introduce these variables at all). I was just trying to follow the "blend in with other code in the file" style guideline.